src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -32,7 +32,7 @@ public void elementsAreUniqueTest(){
 	}
 
 	@Test
-	public void elementsAreUniqueTest2(){
+	public void elementsAreUniqueTest_notUnique_returnFalse(){
 		ArrayList<String> arrayListOf = new ArrayList<String>();
 		arrayListOf.add("hi");
```

Comment:
Indentation problem here? Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.